File: C_Programming/Assignment/Assignment4/rangePerfect.c
#include<stdio.h>
void main()
{
   int num,start,end,sum;
   printf("perfect num are\n");
   for(num=1;num<=1000;num++)
   {
    	sum=0;
    	
    	for(start=1;start<num;start++){
		
    	if(num%start==0)
    	{
    		sum=start+sum;
		}
		}
		if(sum==num)
		{
			printf("%d\n",num);
		}
		
   }
   
}

//output:perfect num are
//6
//28
//496

File: Java_Programs/Assignment4/RangePerfect.java
package Assignment4;

public class RangePerfect {

	public static void main(String[] args) {
		int num=1,start,sum;
		   System.out.printf("perfect num are\n");
		   for(num=1;num<=1000;num++)
		   {
		    	sum=0;
		    	
		    	for(start=1;start<num;start++)
		    	{
				
		    	if(num%start==0)
		    	{
		    		sum=start+sum;
				}
		    	}		
				if(sum==num)
				{
					System.out.printf("%d\n",num);
				}
	}

}
}
